t/dedupe.c

```diff
@@ -37,6 +44,9 @@ struct worker_thread {
 	unsigned long items;
 	unsigned long dupes;
 	int err;
+
+    unsigned long long unique_capacity;
+    struct zlib_ctrl zc;
```

Comment:
Inconsistent indentation here...
```suggestion
	unsigned long items;
	unsigned long dupes;
	int err;

 	unsigned long long unique_capacity;
 	struct zlib_ctrl zc;
```

Reply:
fixed